standards/X509-SVID.md

```diff
@@ -56,7 +56,7 @@ Leaf and signing certificates carry different X.509 properties - some for securi
 ### 4.1. Basic Constraints
 The basic constraints X.509 extension identifies whether the certificate is a signing certificate, as well as the maximum depth of valid certification paths that include this certificate. It is defined in [RFC 5280, section 4.2.1.9][3].
 
-Valid X.509 SVIDs (both leaf and signing certificates) MUST NOT set the `pathLenConstraint` field. Signing certificates MUST set the `CA` field to `true`, and leaf certificates MUST set the `CA` field to `false`.
+SVID signing certificates MUST set the `CA` field to `true` and MUST NOT set the `pathLenConstraint` field. Leaf certificates MUST either omit the basic constraints extension or have an empty basic constraints field (an empty basic constraints field is the only correct encoding of a basic constraint with `CA` set to false.)
```

Comment:
[citation needed].  RFC 5280 says:

> This extension MAY appear as a critical or non-critical extension in end entity certificates.

and allows setting it to false.  What's the reference for it being more correct to omit the basic constraints entirely rather than being explicit?

Reply:
LGTM.